packages/react/src/__tests__/ReactElementJSX-test.js

```diff
@@ -369,8 +369,10 @@ describe('ReactElement.jsx', () => {
     );
   });
 
-  if (require('shared/ReactFeatureFlags').warnAboutSpreadingKeyToJSX) {
-    it('should warn when keys are passed as part of props', () => {
+  it.gate(
+    'warnAboutSpreadingKeyToJSX',
+    'should warn when keys are passed as part of props',
```

Comment:
Hmm this one is interesting because it the gate doesn't actually fail in prod because there are no warnings in prod...

Reply:
I think I'll add support for checking  `__DEV__`, too, even though it's not a flag. That kind fits with how I was planning to eventually add support for gating on a channel.